styles.css

```diff
@@ -11,6 +11,78 @@ body {
   background-color: #e5e5e5;
 }
 
+.toolbar {
+  width: 100vw;
+  height: 50px;
+  display: flex;
+  background-color: white;
+}
+
+/* .toolbar a {
+  text-decoration: none;
+  width: 360px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  padding-top: 24px;
+  text-align: center;
+  color: #6070ff;
+  padding-left: 11%;
+} */
+
```

Comment:
- [ ] Kindly make sure you remove all the commented code lines for [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and clean code as required for each project you make.